docs/content/Getting-Started/user-quickstart.md

```diff
@@ -0,0 +1,106 @@
+
+[![QuickStart test]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml/badge.svg?branch={{config.ks_branch}})]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml)&nbsp;&nbsp;&nbsp;
+
+
+!!! tip "Estimated time to complete this example:" 
+    ~4 minutes (after installing prerequisites)
+
+## How to deploy and use KubeStellar
+
+This guide is intended to show how to (1) quickly bring up a **KubeStellar** environment using helm, (2) install plugins with brew, (3) retrieve your kubeconfig, (4) install KubeStellar Syncers on 2 edge clusters, (5) and deploy an example workload to both edge clusters.
+
+## 0. Pre-reqs
+
+- helm
+- brew
+- kubectl
+- kind clusters (3) 
+   - 1 KubeStellar core cluster (kind-core)
+   - 2 KubeStellar edge clusters (kind-edge-cluster1, kind-edge-cluster2)
+   
+## 1. Install KubeStellar
+
+```
+kubectl cluster-info --context kind-core
```

Comment:
This command does not _set_ the current context. It just uses the given one for the `cluste-info` action. It does not modify any state that affects later commands.

This comment applies to many exhibited command sequences.